src/pages/communication-page/communication.ts

```diff
@@ -348,10 +350,9 @@ export class CommunicationPage extends PracticeableBasePageComponent {
       this.form.controls['radioCtrl'].setValue(true);
     }
 
-    if (this.conductedLanguage !== CommunicationPage.englishLanguage) {
+    if (this.isBookedInWelsh && this.conductedLanguage !== CommunicationPage.englishLanguage) {
```

Comment:
as discussed in person can this be changed to `this.conductedLanguage === CommunicationPage.welshLanguage`?